IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_CableCarrierSegmentTypeCatenaryWire/DocPropertySet.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocPropertySet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="Pset_CableCarrierSegmentTypeCatenaryWire" UniqueId="de4899f4-772e-4fcd-ebf8-9929fa461163" ApplicableType="IfcCableCarrierSegment/CATENARYWIRE" PropertySetType="PSET_TYPEDRIVENOVERRIDE">
+	<Properties>
+		<DocProperty xsi:nil="true" href="ACResistance_2dT1WHiH7Ge2PeqXa73Sp" />
+		<DocProperty xsi:nil="true" href="BreakingLoad_0eJN7IDD15sBhRJRd4TjCU" />
+		<DocProperty xsi:nil="true" href="CatenaryWireType_33lc5OXGT9FexvStyrx03" />
+		<DocProperty xsi:nil="true" href="CoefficientOfLinearExpansion_1f_rsY_rFM9ETgqpq1yXr" />
+		<DocProperty xsi:nil="true" href="CurrentCarryingCapacity_3C4YXmdtnBXOYpmGiU46y8" />
+		<DocProperty xsi:nil="true" href="DCResistance_1iHsZExLz0_hKDmfStAwc4" />
+		<DocProperty xsi:nil="true" href="ElongationPercentage_0HzBKu8tvAFQLOpm2d1Z0" />
+		<DocProperty xsi:nil="true" href="LayRatio_1LJW3Nv3XFAwnt7PZuIg3n" />
+		<DocProperty xsi:nil="true" href="MassPerLength_3veHedEdf31BnVpNC7t3EA" />
+		<DocProperty xsi:nil="true" href="MechanicalTension_0NjLrbe5v3zuECrs8qsUAO" />
+		<DocProperty xsi:nil="true" href="PhysicalDescriptionReference_1KBMgHLx5B9BVSKUJEHSi9" />
+		<DocProperty xsi:nil="true" href="StrandType_3LKqUANgv49RHlqIkKEDH9" />
+		<DocProperty xsi:nil="true" href="TensileStrengthAfterStranded_3uq2zgfMj9p9RWZO_R4dLy" />
+		<DocProperty xsi:nil="true" href="YoungModulus_1x4I40qUqHuO00025QrEV" />
```

Comment:
again material property?

Reply:
I don't think so. Because they are looking at the whole catenary wire which is composed of different material. They want the global mechanical property!